qa/colobus/config

```diff
@@ -730,6 +730,7 @@ group svn.migration {
 group ug.admins {
   num => 104
   path => /var/spool/mlmmj/ug-admins
+  mail => ug-admins@lists.php.net
```

Comment:
This one currently shows up as inactive, wouldn't this change undo that?

Reply:
It was marked inactive in the colobus config, but is actually sort of active -- there were messages last year, [the wiki](https://wiki.php.net/usergroups) still points to it.